addons/mail/static/src/js/thread_windows/abstract_thread_window.js

```diff
@@ -279,14 +279,29 @@ var AbstractThreadWindow = Widget.extend({
      * Render the header of this thread window.
      * This is useful when some information on the header have be updated such
      * as the status or the title of the thread that have changed.
-     *
-     * @private
      */
     renderHeader: function () {
         var options = this._getHeaderRenderingOptions();
         this.$header.html(
             QWeb.render('mail.AbstractThreadWindow.HeaderContent', options));
     },
+    renderOutOfOffice: function () {
```

Comment:
this code should probably be in `ThreadWindow` and not `AbstractThreadWindow` if we follow the template logic